Automattic/crowdsignal-ui
# Create Themes Modal

## Summary

The purpose of this PR is to allow the user to change their current project theme.
On the Document tab from the right sidebar, there will be a `Theme` section that will allow the user to open the theme selection modal.

Depends on: D76409-code

Ref: c/iU4SPLeB-tr

## Test Instructions
* Open a project and, on the right sidebar, under the `Theme` section, click the `Change Theme` button
* You should see the theme selection modal
* The modal should show your current active theme and allow you to change the project theme to one of the available themes

## Screenshots

![image](https://user-images.githubusercontent.com/7811225/157878944-765991de-5fc3-4e5d-a4d5-da12d049af00.png)

![image](https://user-images.githubusercontent.com/7811225/157879015-611c3c6f-878b-442b-86be-4c06bcaf2aa0.png)
